recipes/opencv/meta.yaml

```diff
@@ -0,0 +1,78 @@
+{% set version = "2.4.12" %}
+{% set filename = "opencv2-%s.zip" % version %}
+
+package:
+  name: opencv
+  version: {{ version }}
+
+source:
+  fn: {{ filename }}
+  url: https://github.com/Itseez/opencv/archive/{{ version }}.zip
+  sha1: 2225768a48dbf99426bf77451a433ed432f54d8e
+
+  patches:
+   - osx_rpath.patch  # [osx]
+
+build:
+  number: 0
+  skip: true  # [not py27]
+
+requirements:
+  build:
+   - cmake
+   - curl
+   - numpy x.x
+   - python
+   - eigen 3.*
+   - zlib  # [unix]
+   - jpeg  # [unix]
+   - libtiff  # [unix]
+   - libpng  # [unix]
+
+  run:
+   - python
+   - numpy x.x
+   - zlib  # [unix]
+   - jpeg  # [unix]
+   - libtiff  # [unix]
+   - libpng  # [unix]
```

Comment:
Could we please indent these selectors so that they are inline with one another?